src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2864,7 +2864,10 @@ NeedsDistributedPlanning(Query *queryTree)
 	if (hasLocalRelation && hasDistributedRelation)
 	{
 		ereport(ERROR, (errmsg("cannot plan queries which include both local and "
-							   "distributed relations")));
+							   "distributed relations"),
+						errdetail(
+							"Only exception is INSERT INTO dist_table SELECT ... FROM "
```

Comment:
Hmz, is there a way to only display a relevant error when the query is specifically `INSERT INTO local_table SELECT ... FROM distributed_table ...`?